README.md

```diff
@@ -2,14 +2,17 @@
   <img width="100%" src="https://assets.solidjs.com/banner?type=CLI&background=tiles&project=%20" alt="Solid CLI">
 </p>
 
-# Solid CLI (This is currently very much still in beta)
+> [!WARNING]
+> 🚧 This project is under heavy development and is not yet complete. More features coming soon!
```

Comment:
I'm thinking that I should either remove the "🚧" or keep it and remove the warning callout... it looks a little strange to have both. Thoughts?

Reply:
I agree that having both looks strange. Just removing the "🚧" should be fine I think.